### ISSUE TITLE
Feature: All user photos

### ISSUE THREAD

**Issue by codesworth** [entaq/floq#45]
This is a feature we can add in the future. A way for a user to find all pictures that they have liked or have uploaded from various cliqs all in one place. A sort of personal gallery in the app.

This will be convenient for users to quickly locate a picture they interacted with rather than going through all cliqs in search of a specific photo. Thoughts ?

**Comment by jberkey78** [comment]
Yup, I have designed a version of this already on the account screen... not currently on zeplin. 👍🏻

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by codesworth** [comment]
Alright. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
